content/docs/1.1.0/references/settings.md

```diff
@@ -18,7 +18,7 @@ weight: 1
   - [Automatic Salvage](#automatic-salvage)
   - [Registry Secret](#registry-secret)
   - [Volume Attachment Recovery Policy](#volume-attachment-recovery-policy)
-  - [Pod Deletion Policy When Node Downs](#pod-deletion-policy-when-node-downs)
+  - [Pod Deletion Policy When Node Fails](#pod-deletion-policy-when-node-fails)
```

Comment:
@PhanLe1010 Can we change the setting name to match this?

Reply:
@yasker Sure. It gonna be a lot of changes 😂 Please let me know if I miss something

Reply:
"Pod Deletion Policy when Node is Down" sounds good